source/accounts/generating_keys.rst

```diff
@@ -1,12 +1,11 @@
-#############
-Security Keys
-#############
+##################
+Cryptographic Keys
```

Comment:
i would use the term `SSH Keys` instead, which i think is what most people use, and it's more clear those keys are for SSH authentication.

also, cryptographic keys don't necessarily mean ssh keys.

if you really want to use cryptographic, i would go for `SSH Cryptographic Keys`